ftp-native/src/main/java/org/ballerinalang/stdlib/ftp/util/FTPUtil.java

```diff
@@ -80,12 +80,12 @@ public static String createUrl(BMap config) throws BallerinaFTPException {
                 .getValue();
         int port = extractPortValue(config.getIntValue(StringUtils.fromString(FTPConstants.ENDPOINT_CONFIG_PORT)));
 
-        final BMap secureSocket = config.getMapValue(StringUtils.fromString(
-                FTPConstants.ENDPOINT_CONFIG_SECURE_SOCKET));
+        final BMap auth = config.getMapValue(StringUtils.fromString(
+                FTPConstants.ENDPOINT_CONFIG_AUTH));
         String username = null;
         String password = null;
-        if (secureSocket != null) {
-            final BMap basicAuth = secureSocket.getMapValue(StringUtils.fromString(
+        if (auth != null) {
+            final BMap basicAuth = auth.getMapValue(StringUtils.fromString(
```

Comment:
Better to use `Objects.nonNull(auth)` for null checks.

Reply:
Could not find any reference that would make any improvement. The java doc says `This method exists to be used as a java.util.function.Predicate`. And also Java has implemented that method with `obj != null` which is same as the existing implementation.